chain-core/src/common/merkle_tree.rs

```diff
@@ -113,13 +114,15 @@ impl<T> Tree<T> {
 
 /// Sibling's hash
 #[derive(Debug, PartialEq, Eq, Clone, Encode, Decode)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
```

Comment:
`Serialize` and `Deserialize` shouldn't be added here.

Reply:
i added this for TxOut serde support.
I'll find a way not to use it.